dataRetriever/mock/currentNetworkEpochProviderMock.go

```diff
@@ -0,0 +1,52 @@
+package mock
+
+import (
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+type CurrentNetworkEpochProviderStub struct {
```

Comment:
This is actually a mock as it have functionality that resembles the "real" component. Also, missing comment like `CurrentNetworkEpochProviderStub -`

Reply:
+1 please change the struct name to CurrentNetworkEpochProviderMock

Reply:
Done